nixos/modules/services/networking/syncthing.nix

```diff
@@ -16,6 +16,14 @@ in {
         available on http://127.0.0.1:8384/.
       '';
 
+      guiAddress = mkOption {
+        type = types.string;
```

Comment:
Use `types.str` instead

Reply:
I used types.string for consistency ( it is used in the other options as well )

Reply:
Change all of them to `str` then :)

Reply:
Done